packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -674,8 +674,12 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {
 				// Readout 'more' or 'back' button explicitly.
 				let selectedButton = Spotlight.getCurrent();
 				if (selectedButton === this.mediaControlsNode.querySelector(`.${css.moreButton}`)) {
-					selectedButton.blur();
-					selectedButton.focus();
+					if (this.props.visible) {
+						setTimeout(() => {
```

Comment:
Not sure if it's the right approach to defer focus here, but if it is, you could let `core/Job.start()` handle it.